https.c

```diff
@@ -349,7 +357,11 @@ https_init(const char *useragent, const char *cafile, const char *proxy)
                         return (HTTPS_ERR_LIB);
                 }
         }
+#if OPENSSL_VERSION_NUMBER >= 0x1010000fL
+        if ((ctx->ssl_ctx = SSL_CTX_new(TLS_client_method())) == NULL) {
+#else
         if ((ctx->ssl_ctx = SSL_CTX_new(TLSv1_client_method())) == NULL) {
```

Comment:
outside the scope of your PR, but we need to change this to `SSLv23_client_method`

Reply:
Added this in the `pr_fix_1` branch and merged.

Reply:
Thanks for doing that!